panic: runtime error: invalid memory address or nil pointer dereference

I'm getting the following error, all of the sudden. Using Windows 10 and cmd, worked a few days ago on this shop. Let me know if you need any more info.

panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x1 addr=0x8 pc=0x769f37]

goroutine 1 [running]:
github.com/Shopify/themekit/kit.Environments.GetConfiguration(0xc0422b0e40, 0x85fc30, 0xb, 0x0, 0x0, 0xa52320)
        /Users/timanema/workspace/go/src/github.com/Shopify/themekit/kit/environments.go:71 +0x77
github.com/Shopify/themekit/cmd.generateThemeClients(0xc0420a5bf8, 0x0, 0x0, 0xc0420a5c20, 0x6a838d)
        /Users/timanema/workspace/go/src/github.com/Shopify/themekit/cmd/theme.go:143 +0x4f7
github.com/Shopify/themekit/cmd.glob..func3(0xa53000, 0xa74158, 0x0, 0x0, 0x0, 0x0)
        /Users/timanema/workspace/go/src/github.com/Shopify/themekit/cmd/download.go:25 +0x53
github.com/Shopify/themekit/vendor/github.com/spf13/cobra.(*Command).execute(0xa53000, 0xa74158, 0x0, 0x0, 0xa53000, 0xa74158)
        /Users/timanema/workspace/go/src/github.com/Shopify/themekit/vendor/github.com/spf13/cobra/command.go:599 +0x3f0
github.com/Shopify/themekit/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xa53880, 0x0, 0x0, 0x869800)
        /Users/timanema/workspace/go/src/github.com/Shopify/themekit/vendor/github.com/spf13/cobra/command.go:689 +0x340
github.com/Shopify/themekit/vendor/github.com/spf13/cobra.(*Command).Execute(0xa53880, 0x13, 0x0)
        /Users/timanema/workspace/go/src/github.com/Shopify/themekit/vendor/github.com/spf13/cobra/command.go:648 +0x32
main.main()
        /Users/timanema/workspace/go/src/github.com/Shopify/themekit/cmd/theme/main.go:27 +0x72

--- a/kit/environments.go
+++ b/kit/environments.go
@@ -67,6 +67,12 @@ func (e Environments) GetConfiguration(environmentName string) (*Configuration, 
 	conf, exists := e[environmentName]
 	if !exists {
 		return conf, fmt.Errorf("%s does not exist in this environments list", environmentName)
+	} else if conf == nil {
+		return conf, fmt.Errorf(`Environment %s was found but not defined.
+You may have incorrect indentation in your config.
+Please see %s for examples.`,
+			YellowText(environmentName),
+			BlueText("http://shopify.github.io/themekit/configuration/#config-file"))
 	}
 	conf.Environment = environmentName
 	return conf.compile()

--- a/kit/environments.go
+++ b/kit/environments.go
@@ -68,9 +68,9 @@ func (e Environments) GetConfiguration(environmentName string) (*Configuration, 
 	if !exists {
 		return conf, fmt.Errorf("%s does not exist in this environments list", environmentName)
 	} else if conf == nil {
-		return conf, fmt.Errorf(`Environment %s was found but not defined.
+		return conf, fmt.Errorf(`environment %s was found but not defined.
 You may have incorrect indentation in your config.
-Please see %s for examples.`,
+Please see %s for examples`,
 			YellowText(environmentName),
 			BlueText("http://shopify.github.io/themekit/configuration/#config-file"))
 	}

